docs/adr/20240404-use-domain-driven-design-tactical-patterns.md

```diff
@@ -0,0 +1,41 @@
+# Use Domain-Driven Design Tactical Patterns
+
+- Status: Pending <!-- optional: draft | proposed | rejected | accepted | deprecated | … | superseded by [xxx](yyyymmdd-xxx.md) -->
+- Deciders: {{ TBC }} <!-- optional: list everyone involved in the decision -->
+- Date: 2024-04-04 <!-- optional. YYYY-MM-DD when the decision was last updated. To customize the ordering without relying on Git creation dates and filenames -->
+- Tags: ddd <!-- optional: space and/or comma-separated list of tags -->
+
+Technical Story: {{ TECH STORY }} <!-- optional: description | ticket/issue URL -->
```

Comment:
We need to fill `TECH STORY` up too

Reply:
Done - tech story is linked to #283 and added additional information in the context section